docker/monorail/config.json

```diff
@@ -40,6 +32,13 @@
           "routers": "southbound-api-router"
       }
     ],
+    "taskGraphEndpoint": {
+    "address": "0.0.0.0",
```

Comment:
so the intention is removing southbound-api-router and move to taskGraphEndpoint?